### DISCUSSION TITLE
LOSO cross-validation

### DISCUSSION THREAD

**Discussion by arohl** [pycaret/pycaret#3752] [Q&A]
Hi

I believe I have LOSO CV working in pycaret through setting cv_strategy = LeaveOneGroupOut() 

As expected, the LOSO is being done on the training set in compare_models, and the resultant model is then applied to the test set in evaluate_model. However, I would like to see the LOSO statistics on my complete dataset i.e. I don't want to split my dataset into test and train but make it all training data and then apply LOSO to determine the best model. I will then pass this model to the people who are holding the true test dataset.

Is this possible? Setting the test_data to an empty dataframe in setup doesn't work
